src/tests/ftest/util/file_count_test_base.py

```diff
@@ -33,6 +34,8 @@ def add_containers(self, oclass=None):
             rd_fac = 'rd_fac:{}'.format(str(redundancy_factor))
             properties = (",").join(filter(None, [properties, rd_fac]))
             container.properties.update(properties)
+        if dir_oclass:
+            container.dir_oclass.update(dir_oclass)
```

Comment:
This might be bad if we do, e.g., `container create --oclass S1` because then the DFS root object will be `S1`. Which means ALL client ranks will read root from a single server target.

We should instead do
```
container create --file-oclass S1
```
If you just want to change the file oclass